src/source-pcap-file.c

```diff
@@ -45,6 +45,14 @@
 #include "util-checksum.h"
 #include "util-atomic.h"
 
+//directory support
+#include <dirent.h> // dirent
```

Comment:
these should go into suricata-common.h, properly wrapped in ifdefs and checked for in configure.ac

Reply:
most of them should already be there